### ISSUE TITLE
Close, maximize, and minimize buttons don't look right in cinnamon

### ISSUE THREAD

**Issue by Marcel-Robitaille** [snwh/paper-gtk-theme#286]
I have paper installed on Arch linux (4.3.3-3-ARCH) and Cinnamon 2.8.6. Everything looks great except the close, maximize, and minimize buttons. Here is a screenshot:
![virtualbox_arch linux cinnamon_27_03_2016_11_24_29](https://cloud.githubusercontent.com/assets/8503756/14065808/8ee6e312-f40e-11e5-81c9-6ade96a6850e.png)

**Comment by snwh** [comment]
No, that's how it looks with window managers that don't have CSD support.

**Comment by Marcel-Robitaille** [comment]
Thanks for your support. I thought Cinnamon had CSD support.
